tfx/components/example_gen/utils.py

```diff
@@ -395,7 +454,9 @@ def _create_matching_glob_and_regex(
 
 def _retrieve_latest_span_version(
     uri: Text,
-    split: example_gen_pb2.Input.Split) -> Tuple[Optional[int], Optional[int]]:
+    split: example_gen_pb2.Input.Split,
+    range_config: Optional[range_config_pb2.RangeConfig] = None
```

Comment:
change the name to _get_target_span_version?
as it may not look for latest span & version, if the range-config is present